### ISSUE TITLE
crash caused by deleted videos in playlists 

### ISSUE THREAD

**Issue by youvegottabecrazy** [Protuhj/myvideobuzz#125]
Some deleted videos cause VB to crash. Something changed on Google's end. The deleted item json no longer comes with a contentDetails section.  

Crash happens when vb attempts to construct the videolist, which happens as soon as a playlist is highlighted. That is, if you've got 10 playlists & skip over 9, vb will crash if there's a deleted video in _any_ of those playlists.

Fix: ignore videos with a missing contentDetail section.

**Comment by youvegottabecrazy** [comment]
OK, I'm not entirely sure what I'm doing with pull requests here.  

1.  I've committed the fix this issue (video.brs). 
2.  I have a pull request outstanding, for a linux version of the deploy.bat script.
3.  I go to submit a pull request for this new issue & it looks like this new change has been merged with the outstanding pull request. https://github.com/Protuhj/myvideobuzz/compare/master...youvegottabecrazy:master

Is that how this is supposed to work? If there's a way to do a pull request one specific file I'm just not seeing it.
